symengine/tests/cwrapper/test_cwrapper.c

```diff
@@ -83,11 +83,31 @@ void test_basic() {
     basic_free_heap(y);
 }
 
+void test_complex() {
+    basic e;
+    basic f;
+    char* s;
+    basic_init(e);
+    basic_init(f);
```

Comment:
The error is probably caused by allocating the `e` and `f` here, but never deallocating them. Remember that in C you need to deallocate by hand.